src/shm_manager.cc

```diff
@@ -1,4 +1,5 @@
-// Copyright (c) 2020-2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+// Copyright (c) 2020-2021, NVIDIA CORPORATION & AFFILIATES. All rights
```

Comment:
Remove '(c)' to avoid line wrapping by formatter

Reply:
Fixed.